README.md

```diff
@@ -246,6 +261,43 @@ deduplicate {
   prefix = "consul-template/dedup/"
 }
 
+// This block defines the configuration for exec mode. Please see the exec mode
+// documentation at the bottom of this README for more information on how exec
+// mode operates and the caveats of this mode.
+exec {
+  // This is the command to exec as a child process. There can be only one
+  // command per Consul Template process.
+  command = "/usr/bin/app"
+
+  // This is a random splay to wait before killing the command. The default
+  // value is 0 (no wait), but large clusters should consider setting a splay
+  // value to prevent all child processes from reloading at the same time when
+  // data changes occur. When this value is set to non-zero, Consul Template
+  // will wait a random period of time up to the splay value before reloading
+  // or killing the child process. This can be used to prevent the thundering
+  // herd problem on applications that do not gracefully reload.
+  splay = "5s"
+
+  // This defines the signal that will be sent to the child process when a
+  // change occurs in a watched template. The signal will only be sent after
+  // the process is started, and the process will only be started after all
+  // dependent templates have been rendered at least once. The default value
+  // is "SIGHUP".
+  reload_signal = "SIGUSR1"
```

Comment:
I think you could just set it to a signal that the app doesn't respond to for now. I didn't implement forced-restart. Consul Template just sends the signal to the child process; it does not wait for the child process to respond or take action on that change.